code/game/mecha/components/_component.dm

```diff
@@ -0,0 +1,155 @@
+
+/obj/item/mecha_parts/component
+	name = "mecha component"
+	icon = 'icons/mecha/mech_component.dmi'
+	icon_state = "component"
+	w_class = ITEMSIZE_HUGE
+	origin_tech = list(TECH_DATA = 2, TECH_ENGINEERING = 2)
+
+	var/component_type = null
+
+	var/obj/mecha/chassis = null
+	var/start_damaged = FALSE
+
+	var/emp_resistance = 0	// Amount of emp 'levels' removed.
+
+	var/required_type = null	// List, if it exists. Exosuits meant to use the component.
```

Comment:
If the variable is meant to hold a list (and nothing else), it should be cast as a list

Reply:
I don't know how I missed this.